### PR TITLE
changes for config role

### PR DESCRIPTION
**Pull request by muhammad-ammar**
@yro @muzaffaryousaf @mushtaqak @Qubad786 These are the changes required to make ansible role working. Please review the changes.

### CODE REVIEW

**Comment by muzaffaryousaf** [comment]
@muhammad-ammar can you rebase this PR ? I'm thinking to roll this out along with credentials work. What do you think ?

**Comment by muzaffaryousaf** [comment]
@muhammad-ammar can we split this bigger PR into smaller one. It would be easier to review and track any issues more easily.
Or at-least at-least i would split out https://github.com/edx/edx-video-pipeline/pull/33/commits/c9c30655af5e4955b216c99762fd829a44e539c6 into separate PR.

**Comment by muzaffaryousaf** [comment]
I would also extract this commit into separate PR https://github.com/edx/edx-video-pipeline/pull/33/commits/cc94b0d42345e29e3e7927c6d1f957556e41c9b1

File: VEDA/tests/test_utils.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            org,
+            expected_org
+        )
+
+    def test_get_config(self):
+        """
+        Tests that utils.get_config works as expected.
+        """
+        config = utils.get_config()
+        self.assertNotEqual(config, {})
```
Comment (Qubad786) [inline VEDA/tests/test_utils.py]: why is it empty? isn't it loading `instance_config.yaml`?

File: VEDA/tests/test_utils.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        """
+        Tests that utils.extract_course_org works as expected.
+        """
+        org = utils.extract_course_org(course_id)
+        self.assertEqual(
+            org,
+            expected_org
+        )
+
+    def test_get_config(self):
```
Comment (Qubad786) [inline VEDA/tests/test_utils.py]: need to cover YamlError case and os.environ case.

File: VEDA/tests/test_utils.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
+"""
```
Comment (Qubad786) [inline VEDA/tests/test_utils.py]: @muhammad-ammar I think we need to add tests for logger config as well?

File: requirements.txt [hunk 4]
```diff
@@ -13,4 +12,7 @@ pyyaml
 requests==2.18.1
 celery==3.1.18
 pysrt==1.1.1
-edx-opaque-keys==0.4
\ No newline at end of file
+edx-opaque-keys==0.4
```
Comment (Qubad786) [inline requirements.txt]: can we sort them in alphabetical order?

File: instance_config.yaml [hunk 5]
```diff
@@ -13,11 +13,11 @@
 DATABASES:
     default:
         ENGINE: 'django.db.backends.mysql'
```
Comment (Qubad786) [inline instance_config.yaml]: Do we still need to set up an sql db named 'pipeline' manually while setting up the local env?

**Comment by muzaffaryousaf** [comment]
@yro can you please give it a look.

File: VEDA/settings.py [hunk 6]
```diff
@@ -44,8 +44,9 @@
 USE_L10N = True
 USE_TZ = True
 
+
 # Absolute filesystem path to the directory that will hold user-uploaded files.
-STATIC_ROOT = os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))), "static")
+STATIC_ROOT = STATIC_ROOT_PATH
 STATIC_URL = '/static/'
```
Comment (yro) [inline VEDA/settings.py]: Nice, thank you.

**Pull request merged**